scripts/YouTubeScraper.py

```diff
@@ -83,6 +83,13 @@ def search_comments_by_name(query: str, max_results=100):
 
 if __name__ == "__main__":
     # You can search by video name, or video id directly.
-    results = search_comments_by_name("Video Name", 10)
+    results = search_comments_by_name("Racist Karens KNOCKED OUT COLD And Getting Instant Karma! | BIG KARMA", 500)
+    results = search_comments_by_name("Full Debate: Biden and Trump in the First 2024 Presidential Debate | WSJ", 1000)
+    results = search_comments_by_name("\"You're Glorifying Obesity!\" Piers Morgan Interviews TikTok 'Fat Influencer'", 1000)
+    results = search_comments_by_name("RiceGum - Its EveryNight Sis feat. Alissa Violet (Official Music Video)", 1000)
+    results = search_comments_by_name("Worst Youtuber Apology Ever", 1000)
+    results = search_comments_by_name("Konstantin Kisin: Woke Culture HAS Gone Too Far - 7/8 | Oxford Union", 1000)
```

Comment:
We can store the names in a ReadMe rather than having then in the youtube scaper